app/controllers/admin/listing_shapes_controller.rb

```diff
@@ -222,6 +220,7 @@ def common_locals(form:, count:, process_summary:, available_locs:)
       uneditable_fields: uneditable_fields(process_summary, form[:author_is_seller]),
       shape: FormViewLayer.shape_to_locals(form),
       count: count,
+      harmony_in_use: APP_CONFIG.harmony_api_in_use.to_s == "true",
```

Comment:
I don't think this should be here, as Harmony is no longer used at all and this configuration variable was removed. Is there a reason why it was added here?